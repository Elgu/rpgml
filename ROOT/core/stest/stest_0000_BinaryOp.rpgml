PRINT( ( 1 + 2 ) + "\n" );
PRINT( ( 2.5 + 1 ) + "\n" );
PRINT( "foo" + "\n" );

Output i = counter();

print( i + "\n" );

if( false )
{
  Output a1 = [ 3, 4, -3 ];
  Output s1 = [ "foo", "bar", "baz" ];
  print( a1 + s1 );
  print( "\n" );
}

exit( i == 1 );

