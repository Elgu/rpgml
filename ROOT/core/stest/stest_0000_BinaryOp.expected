3
3.5
foo
0
1
